pyproject.toml

```diff
@@ -29,7 +29,7 @@ dependencies = [
     "ga4gh.vrs ~=2.0.0a10",
     "biocommons.seqrepo",
     "gene-normalizer ==0.4.0",
-    "cool-seq-tool ~=0.7.1",
+    "cool-seq-tool ~=0.8.0",
```

Comment:
do the tests still pass after making this change? I wasn't sure if it would impact any since I'm not sure if we have any weird capitalization stuff, but I just wanted to check!

Reply:
Good call, I can check to see if the tests still pass

Reply:
Looks like the tests are still passing on my end